File: src/main/java/sage/transfer/Stream.java
package sage.transfer;

import java.util.ArrayList;
import java.util.Collection;
import java.util.List;

public class Stream {
  private List<Item> items;

  public Stream() {
    items = new ArrayList<>();
  }

  public Stream(List<? extends Item> _items) {
    items = new ArrayList<>(_items);
  }

  public void add(Item item) {
    getItems().add(item);
  }

  public void addAll(Collection<? extends Item> _items) {
    getItems().addAll(_items);
  }

  public List<Item> getItems() {
    return items;
  }

  @Override
  public String toString() {
    return "Stream tweets count: " + items.size();
  }
}

File: src/main/kotlin/sage/transfer/Stream.kt
package sage.transfer

import sage.annotation.KotlinNoArg
import java.util.ArrayList

@KotlinNoArg
class Stream {
  val items: MutableList<Item> = ArrayList()

  constructor(_items: List<Item>) {
    items += _items
  }

  override fun toString(): String {
    return "Stream tweets count: " + items.size
  }
}
